specification/xpack/usage/types.ts

```diff
@@ -0,0 +1,428 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Phases } from '@ilm/_types/Phases'
+import { Dictionary } from '@spec_utils/Dictionary'
+import { ByteSize, EmptyObject, Field, Name } from '@_types/common'
+import { Job, JobStatistics } from '@_types/ml/Job'
+import { integer, long, uint, ulong } from '@_types/Numeric'
+import { Statistics } from '@_types/slm/Statistics'
+
+export class Usage {
+  available: boolean
+  enabled: boolean
+}
+
+export class SecurityFeatureToggle {
+  enabled: boolean
+}
+
+export class AlertingExecution {
+  actions: Dictionary<string, ExecutionAction>
+}
+
+export class AlertingInput {
+  input: Dictionary<string, UsageCount>
+  trigger: Dictionary<string, UsageCount>
+}
+
+export class AnalyticsStatsUsage {
```

Comment:
There is no need to name all of this types `*Usage`, as those are already defined inside the `usage` folder.
You should either drop the `Usage` from the name, or move all of these definitions to the `xpack/_types` folder.

Reply:
Right! Will rename